config/menus/options.cfg

```diff
@@ -421,6 +421,12 @@ options_statistics = [
         ui_strut 1
         ui_radiobutton2 "more verbose" showfps 3
     ]
+    ui_text "On-Screen System Time:"
```

Comment:
Include *where* it will pop up, please.

Reply:
That would be inconsistent looking at the rest of the interface, e.g. On-Screen FPS